README.md

```diff
@@ -8,17 +8,18 @@ To quickly learn the basics of running cleanlab on your own data, we recommend f
+| 4   | [simplifying_confident_learning](4_simplifying_confident_learning/simplifying_confident_learning.ipynb) | Straightforward implementation of Confident Learning algorithm with raw numpy code.                                                                                                                                                                                                                    |
+| 5   | [visualizing_confident_learning](5_visualizing_confident_learning/visualizing_confident_learning.ipynb)                   | See how cleanlab estimates parameters of the label error distribution (noise matrix).                                                                                                                                                                                                                   |
+| 6   | [cifar10_cnn_coteaching](6_cifar10_cnn_coteaching)                                               | Train a [Convolutional Neural Network](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/experimental/cifar_cnn.py) on noisily labeled Cifar10 image data using cleanlab with [coteaching](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/experimental/coteaching.py) |
+| 7   | [mnist_cnn](7_mnist_cnn/label_errors_mnist_train_cnn.ipynb)                                                                         | Finding label errors in MNIST image data with a [Convolutional Neural Network](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/experimental/mnist_pytorch.py)                                                                                          |
+| 8   | [amazon_reviews_fasttext](8_amazon_reviews_fasttext/amazon_pyx.ipynb)                                             | Demonstrate use of a cleanlab-compatible  [FastText model](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/experimental/fasttext.py)                                                                                                    |
+| 9   | [outlier_detection_cifar10](9_outlier_detection_cifar10/outlier_detection.ipynb)                                             | Train AutoML for image classification and use it to detect out-of-distribution images.                                                                                                 |
+| 10  | [cifar10_multiannotator](10_cifar10_multiannotator/multiannotator_labels.ipynb)                                             | Iteratively improve consensus labels and trained classifier from data labeled by mulitple annotators.                                                            |
+| 11  | [entity_recognition](11_entity_recognition/entity_recognition_training.ipynb)                                             | Train Transformer model  for Named Entity Recognition and produce out-of-sample `pred_probs` for cleanlab.token_classification.      
+
 
 ## Instructions
 
```

Comment:
add section to instructions that describes: 

Instead of installing the requirements for all examples simultaneously via `pip install -r requirements.txt`, you can also install only the requirements for a particular example by executing this command inside of the corresponding folder. This will require that you have installed cleanlab (`pip install cleanlab`), and some examples may require you to have the latest developer version of cleanlab from github (`pip install git+https://github.com/cleanlab/cleanlab.git`).